packages/recommend-core/src/getFrequentlyBoughtTogether.ts

```diff
@@ -29,6 +29,10 @@ export function getFrequentlyBoughtTogether<TObject>({
 
   recommendClient.addAlgoliaAgent('recommend-core', version);
 
+  if (queries.length === 0) {
+    return Promise.resolve({ recommendations: [] });
+  }
```

Comment:
I guess that's fine but there's no way back after we release it this way. What used to be an API error now fails silently.